test/e2e/scheduler_predicates.go

```diff
@@ -110,6 +110,22 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"reason":                   "FailedScheduling",
 		}.AsSelector())
 	expectNoError(err)
+	// If we failed to find event with a capitalized first letter of reason
+	// try looking for one starting with a small one for backward compatibility.
+	// If we don't do it we end up in #158061.
+	// TODO: remove this block when we don't care about supporting v1.0 too much.
```

Comment:
We never done that. At least I haven't seen this pattern. I'd rather leave it as it is, it's not worth having separate issue for this kind of cleanup. Someone will do this when changing this test during scheduler refactoring.